.github/scripts/detect_unchanged_ids.py

```diff
@@ -0,0 +1,107 @@
+from fluent.syntax import parse
+import os
+import sys
+import argparse
+
+try:
+    from compare_locales import parser
+except ImportError as e:
+    print("FATAL: make sure that dependencies are installed")
+    print(e)
+    sys.exit(1)
+
+def get_file_list(path):
+    """Get the list of supported files."""
+    file_list = []
+    for root, dirs, files in os.walk(path, followlinks=True):
+        for file in files:
+            if file.endswith(".ftl"):
+                file_list.append(os.path.join(root, file))
+    file_list.sort()
+
+    return file_list
+
+
+def generate_dict(file_name):
+    file_parser = parser.getParser(".ftl")
+    file_parser.readFile(file_name)
+    try:
+        entities = file_parser.parse()
+        entity_dict = {}
+        for entity in entities:
+            # Ignore Junk
+            if isinstance(entity, parser.Junk):
+                continue
+            string_id = f"{entity}"
+            if entity.raw_val is not None:
+                entity_dict[string_id] = entity.raw_val
+            # Store attributes
+            for attribute in entity.attributes:
+                attr_string_id = f"{entity}.{attribute}"
+                entity_dict[attr_string_id] = attribute.raw_val
+        return entity_dict
+    except Exception as e:
+        print(f"Error parsing file: {file_name}")
+        print(e)        
+
+
+def find_changed_ids(file_old, file_new):
+    dict_old = generate_dict(file_old)
+    dict_new = generate_dict(file_new)
+    return {
+        key: {'before': dict_old[key], 'after': dict_new[key]}
+        for key in dict_old.keys()
+        if key in dict_new and dict_old[key] != dict_new[key]
+    }
+
+
+def main():
+    arg_parser = argparse.ArgumentParser()
+    arg_parser.add_argument("dir_old")
```

Comment:
Please make these named arguments and add some description, at least it's impossible to swap the files by mistake, e.g.

```
    parser.add_argument(
        "--old",
        required=True,
        help="Folder with the previous version of the file",
    )
```

Reply:
Fixed